transforms/em-to-ed/index.js

```diff
@@ -68,13 +70,82 @@ function stripId(j, source) {
       key: {
         name: 'id',
       },
+      value: {
```

Comment:
This makes sure we don't nuke any old objects with the `id` key, just those like `id: attr()`.